data/human/culture independents.txt

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2015 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# "Missions" which add culture events to the spaceport.
+
+# Courtesy of ES Celestia
+mission "ZZCU Tarazed Ship Christening"
+	minor
+	source "Wayfarer"
+	to offer
+		random == 1
+		has "main plot complete"
+	on offer
+		conversation
```

Comment:
I'll have you know that this is a single-space-after-periods repository. I'm not going to create suggestions for every single line, but that needs changed.